src/rez/resolved_context.py

```diff
@@ -23,6 +23,7 @@
 from rez.vendor import yaml
```

Comment:
@elrond79, did you know that by setting 'config.catch_rex_errors' to False, you can see the trace on rex errors also?